store/rootmulti/store.go

```diff
@@ -60,6 +60,7 @@ type Store struct {
 	db                  corestore.KVStoreWithBatch
 	logger              iavltree.Logger
 	lastCommitInfo      *types.CommitInfo
+	lastCommitInfoMut   sync.RWMutex
```

Comment:
Instead of adding a mutex, can we make the `lastCommitInfo` an atomic pointer, if the content of `lastCommitInfo` is not modified concurrently, which seems to be the case?

Reply:
yea seems reasonable, let me try